src/pkg/cli/debug.go

```diff
@@ -74,10 +74,16 @@ func Debug(ctx context.Context, c client.FabricClient, p client.Provider, etag t
 		Services: failedServices,
 		Project:  project.Name,
 	}
-	err := p.Query(ctx, &req)
-	if err != nil {
-		return err
+
+	if etag != "" {
```

Comment:
this can be a `switch-case` with a `default:` that panics or returns an error because there's nothing to debug.